src/Particular.CodeRules/Particular.ruleset

```diff
@@ -1,6 +1,566 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="Particular Software Rule Set" Description=" CodeAnalysis Rules for Particular Software Projects" ToolsVersion="14.0">
+<RuleSet Name="Particular Software Rule Set" Description=" CodeAnalysis Rules for Particular Software Projects" ToolsVersion="16.0">
```

Comment:
Apparently there is also a rule set that disables all the rules 

https://docs.microsoft.com/en-us/visualstudio/code-quality/analyzer-rule-sets?view=vs-2019#use-analyzer-rule-sets

Maybe you can then enable certain rules again. I haven't looked deeply into it so hopefully this is helpful and not a drive by comment.

Reply:
@danielmarbach I've searched for a global opt-in approach instead of opt-out but I could not find it. I've read the document that you've linked but there I only see how to load a specific ruleset which is what we do:
```xml
<PropertyGroup>
  ...
  <CodeAnalysisRuleSet>Particular.ruleset</CodeAnalysisRuleSet>
</PropertyGroup>
```

Did I mis something on that page?

Reply:
It looks like you would have to reference the ruleset to disable with absolute paths https://github.com/dotnet/roslyn-analyzers/issues/943. So that is probably not really something useful then. So I guess how you do it today is the only option...